### PR TITLE
Init Azure.ResourceManager.PolicyInsights

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Review by Yao725 — APPROVED** [review]

File: sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      delete $['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/remediations/{remediationName}/listDeployments'];
+      delete $['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/remediations/{remediationName}/cancel'];
+  - from: attestations.json
+    where: $.paths
+    transform: >
+      delete $['/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/attestations'];
+      delete $['/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/attestations/{attestationName}'];
+      delete $['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/attestations'];
+      delete $['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/attestations/{attestationName}'];
+  - from: remediations.json
```
Comment (ArcturusZhang) [inline sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md:81]: Could we also add a comment here saying that below directives are resolving the duplicate schema issues?

**Review by ArcturusZhang — APPROVED** [review]

File: sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md:82]: fixed

**Pull request merged**